### DISCUSSION TITLE
rb.fetch_by_column 怎么查询多个字段

### DISCUSSION THREAD

**Discussion by mpv945** [rbatis/rbatis#262] [General]
就单表简单查询，想用内置的宏写了。但是示例只有单字段，如果多字段要查询，中间使用and或or 怎么操作？还是多fetch_by_column只支持单个条件查询

**Comment by zhuxiujia** [comment]
> 就单表简单查询，想用内置的宏写了。但是示例只有单字段，如果多字段要查询，中间使用and或or 怎么操作？还是多fetch_by_column只支持单个条件查询

可以使用py_sql或者html_sql 宏获取bson数据（任意的colume）
文档：https://rbatis.github.io/rbatis.io/#/?id=pysql%e8%af%ad%e6%b3%95
例子：https://github.com/rbatis/rbatis/blob/master/example/src/macro_proc_pysql.rs
